ads/google/a4a/utils.js

```diff
@@ -143,45 +143,46 @@ export function googleAdUrl(
   // TODO: Maybe add checks in case these promises fail.
   /** @const {!Promise<string>} */
   const referrerPromise = viewerForDoc(a4a.getAmpDoc()).getReferrerUrl();
-  return getAdCid(a4a).then(clientId => referrerPromise.then(referrer => {
-    const adElement = a4a.element;
-    window['ampAdGoogleIfiCounter'] = window['ampAdGoogleIfiCounter'] || 1;
-    const slotNumber = window['ampAdGoogleIfiCounter']++;
-    const win = a4a.win;
-    const documentInfo = documentInfoForDoc(adElement);
+  return getClientScopedAdCid(a4a.getAmpDoc(), a4a.win, 'AMP_ECID_GOOGLE').then(
```

Comment:
This is a util specifically for google though, so only used by adsense and doubleclick